compiler/hash-ast/src/error.rs

```diff
@@ -33,3 +33,15 @@ impl From<(io::Error, PathBuf)> for ParseError {
         }
     }
 }
+
+impl From<ParseIntError> for ParseError {
+    fn from(_: ParseIntError) -> Self {
+        todo!()
+    }
+}
+
+impl From<ParseFloatError> for ParseError {
+    fn from(_: ParseFloatError) -> Self {
+        todo!()
+    }
+}
```

Comment:
So I guess here we need a location and source of the file?

Reply:
Wtf, can you believe that I didn't actually add these intentionally... I must have accidentally pressed some language server code action.

Reply:
I see, I guess that's why they teach you to code review at computer school.

Reply:
Yep 😅

Reply:
Never mind, it turns out that these are necessary to implement the modified int literal parsing. I think you must have added them?